Assets/Scripts/PlayerScripts/HealLife.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class HealLife : MonoBehaviour
+{
+    private HealthController healthController = default;
+    private PlayerController _playerController = default;
+    private InputAction _inputHeal = default;
+    void Start()
+    {
+        _playerController = new PlayerController();
+        _playerController.Enable();
+        _inputHeal = _playerController.Land.Heal;
+        _inputHeal.Enable();
+        _inputHeal.started += _ => Healing();
+    }
+    private void OnDestroy()
```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah seriously? I just told you

Reply:
que ya est{a viejo loco aprenda a lerr

Reply:
reprobado